scss/_tonality.scss

```diff
@@ -21,15 +22,15 @@
 		.db-ui-#{nth($styles, $i + 1)} {
 			@extend %db-scaling-#{nth($styles, $i + 1)};
 
-			[data-size="large"] {
-				@extend %db-#{nth($styles, $i + 1)}-body-lg;
-			}
-
-			:not(small),
+			*,
```

Comment:
CSS Variables, and the defined `font-size` and `line-height` are inherited, so I'm wondering whether we write it like this instead:
```SCSS
			// General styling
			@extend %db-#{nth($styles, $i + 1)}-body-md;
			
			[data-size="medium"] {
				@extend %db-#{nth($styles, $i + 1)}-body-md;
			}
```

Reply:
This is not working...
I had this before, have you tried this?